### DISCUSSION TITLE
Will there be any updates to tables in the next release?

### DISCUSSION THREAD

**Discussion by GraphicOwls** [notaapp/talk#340] [Questions]
Will there be any updates to tables in the next release? I wish I didn't have to toggle the preview to view tables.

**Comment by astoilkov** [comment]
Hey @GraphicOwls. No, the next release won't include any improvements to tables. Sorry (it's always sad to disappoint a user 😥).

**Comment by GraphicOwls** [comment]
Aw that's a bummer. Hopefully not too long next year. Curious, what's the real point of preview mode? We can't share anything publicly and most content already looks great inline in the editor, so why waste time on the preview mode? I like when you see the markdown when your cursor is on the line you're editing, and when you leave that line, it renders it perfectly. So much nicer, and Nota already does this with most things in the editor currently.

**Reply by astoilkov** [reply]
Yeah, I agree. We want to get rid of the preview when the editor is good enough.

What might be not obvious is that making a good editor is a very hard thing to do. It's just a hard engineering problem.
